libraries/Microsoft.Bot.Builder.AI.QnA/QnAMakerExtension.cs

```diff
@@ -0,0 +1,54 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.IO;
+using System.Text.RegularExpressions;
+using Microsoft.Extensions.Configuration;
+
+namespace Microsoft.Bot.Builder.AI.QnA
+{
+    /// <summary>
+    /// Extension methods for QnA.
+    /// </summary>
+    public static class QnAMakerExtension
```

Comment:
Please rename the class to IConfigurationBuilderExtensions.

Reply:
I follow the name of luisExtension.cs. Do I have to rename it?

Reply:
Yes please, the luisExtensions file is wrongly named too. In general, extension methods should be placed in files called _TypeName_Extensions.cs, there `TypeName`, is the name of the type that the extension methods apply to. This makes it easier to spot them in the source tree. 

We haven't followed this naming convention consistently but we should.